Src/UUIDNext/UUIDNext.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFramework>netstandard2.1</TargetFramework>
+		<TargetFrameworks>netstandard2.1;netstandard2.0</TargetFrameworks>
```

Comment:
I checked [.NET Standard's version support table](https://dotnet.microsoft.com/en-us/platform/dotnet-standard) and it seems that targeting .NET standard 2.0 in addition to 2.1 only add support for .NET framework and older version of mono/xamarin/unity.
As I don't want people building modern .NET App to accidentally use the slower .NET Standard 2.0, I think it's safer to not target multiple .NET Standard versions.
Can you change the TargeFrameworks to `netstandard2.1;net472` ?

Reply:
Changed this to now target net472 as you suggested

Reply:
@mareek I think there might be a way to handle this but also support wider set of frameworks and standards. I have not done this before but I can see some other OSS projects do it.
I might look and see if I can submit a pull request later with those changes. (or if anyone else do it if they have time/capacity right now)
fyi @rocklan

Reply:
@rbakhshi If you find a way to support .NET Standard 2.0 without impacting perf/memory for .NET standard 2.1 I will gladly merge it